README.md

```diff
@@ -22,108 +22,113 @@ Our React.js SDK is distributed via [npm](https://www.npmjs.com/), and can be in
 npm i @evervault/react
 ```
 
-## Setup
+## Initialization
 
-To make Evervault available for use in your app, use an `EvervaultProvider` component as a provider for your app.
+Once installed, initialize the React.js SDK with your team's unique ID found in the [Settings](https://app.evervault.com/settings). Use an `EvervaultProvider` component as a provider for your app.
 
 ```javascript
-import { EvervaultProvider } from '@evervault/react';
+import { EvervaultProvider } from "@evervault/react";
 
 export const App = () => {
-  <EvervaultProvider teamId={'<YOUR-TEAM-ID>'}>
+  <EvervaultProvider teamId={"<YOUR-TEAM-ID>"}>
     <ChildComponent />
-  </EvervaultProvider>
-}
+  </EvervaultProvider>;
+};
 ```
```

Comment:
Pretty sure this code is invalid (it was already), missing the implicit return, like so:

```javascript
import { EvervaultProvider } from "@evervault/react";
export const App = () => (
  <EvervaultProvider teamId={"<YOUR-TEAM-ID>"}>
    <ChildComponent />
  </EvervaultProvider>;
);
```